docs/decisions/021-node-version.md

```diff
@@ -8,7 +8,7 @@ Status: accepted
 
 Node.js has a regular release cycle which is documented in the
 [release schedule](https://nodejs.org/en/about/releases/). At the time of this
-writing, there are 3 stable maintained releases: 16, 18, and 20. I'll refer you
+writing, there are 2 stable maintained releases: 18, and 20. I'll refer you
```

Comment:
This says "at the time of this writing" for a reason. This is meant to be a record of historical decisions and will not be updated.

Thanks though!